repos/system_upgrade/el7toel8/actors/persistentnetnames/actor.py

```diff
@@ -0,0 +1,17 @@
+import os
+import pyudev
+
+from leapp.libraries.common import *
+from leapp.actors import Actor
+from leapp.models import PCIAddress, Interface, PersistentNetNamesFacts
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+class PersistentNetNames(Actor):
+    name = 'persistentnetnames'
+    description = 'Gather current network naming configuration'
+    consumes = ()
+    produces = (PersistentNetNamesFacts)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        self.produce(PersistentNetNamesFacts(interfaces = list(persistentnetnames.interfaces())))
```

Comment:
```suggestion
        self.produce(PersistentNetNamesFacts(interfaces=list(persistentnetnames.interfaces())))
```
In this case, you should omit whitespaces around the `=`.